src/commands/operator/index.ts

```diff
@@ -153,13 +120,32 @@ export default class Operator extends Command {
 
     this.operatorStatus.address = userWallet.address.toLowerCase()
 
+    this.networkMonitor.exitCallback = this.exitCallback.bind(this)
+
     CliUx.ux.action.start(`Starting operator in mode: ${OperatorMode[this.operatorMode]}`)
     await this.networkMonitor.run(true, undefined, this.filterBuilder)
     CliUx.ux.action.stop('🚀')
 
+    // check if file exists
+    if (await fs.pathExists(this.jobsFile)) {
+      // if file exists, need to add it to list of jobs to process
+      this.operatorJobs = (await fs.readJson(this.jobsFile)) as {[key: string]: OperatorJob}
+      // need to check each job and make sure it's still valid
+      for (const jobHash of Object.keys(this.operatorJobs)) {
+        this.operatorJobs[jobHash].gasLimit = BigNumber.from(this.operatorJobs[jobHash].gasLimit)
+        this.operatorJobs[jobHash].gasPrice = BigNumber.from(this.operatorJobs[jobHash].gasPrice)
+        this.operatorJobs[jobHash].jobDetails.startTimestamp = BigNumber.from(
+          this.operatorJobs[jobHash].jobDetails.startTimestamp,
+        )
+        // if job is still valid, it will stay in object, otherwise it will be removed
+        // eslint-disable-next-line no-await-in-loop
+        await this.checkJobStatus(jobHash)
+      }
+    }
+
     for (const network of this.networkMonitor.networks) {
       // instantiate all network operator job watchers
-      this.processOperatorJobs(network)
+      setTimeout(this.processOperatorJobs.bind(this, network), 60_000)
```

Comment:
This is a temporary solution to allow Network Monitor to gather enough block data about gas prices to better know when it is possible and profitable to submit jobs.

Reply:
Later this will be removed as I finalize the internal Network Monitor logic that monitors latest blocks while the command catches up to missed blocks.